metrix-integration/src/main/groovy/com/powsybl/metrix/integration/MetrixDslDataLoader.groovy

```diff
@@ -108,7 +110,15 @@ class MetrixDslDataLoader {
 
     static void evaluate(GroovyCodeSource dslSrc, Binding binding) {
         def config = createCompilerConfig()
+
+        // Add a check on thread interruption in every loop (for, while) in the script
+        config.addCompilationCustomizers(new ASTTransformationCustomizer(ThreadInterrupt.class))
```

Comment:
Could you add this in `createCompilerConfig`, similarly to what's done in `TimeSeriesDslLoader.groovy`?